client/src/components/ViewDocument.js

```diff
@@ -59,7 +59,7 @@ export class ViewDocument extends Component {
     this.state.translation
       ? (translationContent = this.state.translation.content)
       : (translationContent = "");
-
+    //console.log(this.state.document.content);
```

Comment:
can u please remove commented out console log

Reply:
I removed it